__tests__/app.test.js

```diff
@@ -223,16 +223,32 @@ describe('PATCH /api/articles/:article_id', () => {
     .send(updateObj)
     .expect(200)
     .then(({ body }) => {
-      expect(body.article).toEqual(expectedOutput);
+      expect(body.article).toEqual(expectedOutput[0]);
```

Comment:
Make the expectedOutput an object rather than an array with an object inside it so you don't have to check [0]